task_manager:
  ros__parameters:
    # Motion Planner 配置
    use_motion_planner: false         # 是否使用motion_planner
    approach_distance: 0.1            # 接近距离 (m)
    retreat_distance: 0.05            # 后退距离 (m)
    max_trajectory_points: 100        # 最大轨迹点数
    
    # 原有配置...
    pre_grasp_height: 0.05
    grasp_height_offset: 0.02
    gripper_length: 0.1
    lift_height: 0.05
    place_offset_x: 0.0
    place_offset_y: 0.2
    place_offset_z: 0.0
    wait_time_at_object: 2.0
    home_position_x: 0.0
    home_position_y: 0.0
    home_position_z: 0.4
    home_orientation_roll: 0.0
    home_orientation_pitch: 0.0
    home_orientation_yaw: 0.0